Evaluate-ModelarDB-Changes/main.py

```diff
@@ -110,6 +97,16 @@ def start_modelardbd(modelardb_cargo_toml, data_folder):
     return process
 
 
+def errors_occured(output_stream):
```

Comment:
Minor but I think it should be `occurred` instead.

Reply:
Fixed.